submitted_models/costar_husky_sensor_config_2/model.sdf

```diff
@@ -0,0 +1,696 @@
+      <kinematic>0</kinematic>
+      <sensor name="camera_front" type="rgbd_camera">
+        <camera name="camera_front">
+            <horizontal_fov>1.0472</horizontal_fov>
+            <lens>
+                <intrinsics>
+                    <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+                    <fx>277.1</fx>
+                    <fy>277.1</fy>
+                    <!-- cx = ( width + 1 ) / 2 -->
+                    <cx>160.5</cx>
+                    <!-- cy = ( height + 1 ) / 2 -->
+                    <cy>120.5</cy>
+                    <s>0</s>
+                </intrinsics>
+            </lens>
+            <distortion>
+                <k1>0.0</k1>
+                <k2>0.0</k2>
+                <k3>0.0</k3>
+                <p1>0.0</p1>
+                <p2>0.0</p2>
+                <center>0.5 0.5</center>
+            </distortion>
+            <image>
+                <width>320</width>
+                <height>240</height>
+                <format>R8G8B8</format>
+            </image>
+            <clip>
+                <near>0.01</near>
+                <far>300</far>
+            </clip>
+            <depth_camera>
+              <clip>
+                <near>0.1</near>
+                <far>10</far>
+              </clip>
+            </depth_camera>
+            <noise>
+                <type>gaussian</type>
+                <mean>0</mean>
+                <stddev>0.007</stddev>
+            </noise>
+        </camera>
+        <always_on>1</always_on>
+        <update_rate>20</update_rate>
+        <visualize>0</visualize>
+        <pose frame="">0.438 0 0.272 0 -0 0</pose>
+      </sensor>
+      <gravity>1</gravity>
+      <velocity_decay/>
+      <sensor name="front_cliff_laser" type="gpu_ray">
+          <pose>0.435 0 0.235 0 -2.4434589 0</pose>
+          <update_rate>50</update_rate>
+          <lidar>
+              <scan>
+                  <horizontal>
+                      <samples>1</samples>
+                      <resolution>1</resolution>
+                      <min_angle>3.141592653589793</min_angle>
+                      <max_angle>3.141592653589793</max_angle>
+                  </horizontal>
+              </scan>
+              <range>
+                  <min>0.04</min>
+                  <max>40</max>
+                  <resolution>0.01</resolution>
+              </range>
+              <noise>
+                  <type>gaussian</type>
+                  <mean>0</mean>
+                  <stddev>0.01</stddev>
+              </noise>
+          </lidar>
+      </sensor>
+      <gravity>1</gravity>
+      <velocity_decay/>
+      <sensor name="front_laser" type="gpu_lidar">
+        <visualize>0</visualize>
+        <update_rate>10</update_rate>
+        <lidar>
+          <scan>
+            <horizontal>
+              <!-- Resolution can't be above 16k, see
+              https://bitbucket.org/ignitionrobotics/ign-sensors/issues/8 -->
+              <samples>10000</samples>
```

Comment:
This (along with all other places where 3D lidar is used) should be corrected to have correct horizontal resolution matching physical VLP-16 (At 10 Hz it would be 1800 horizontal samples)

http://www.mapix.com/wp-content/uploads/2018/07/63-9229_Rev-H_Puck-_Datasheet_Web-1.pdf